redhat/cassandra.spec

```diff
@@ -85,28 +85,25 @@ rm -f tools/bin/*.bat
 rm -f tools/bin/cassandra.in.sh
 
 # copy default configs
-cp -pr conf/* %{buildroot}/%{_sysconfdir}/%{username}/default.conf/
+cp -pr conf/* %{buildroot}/%{_sysconfdir}/%{cassandraX}/default.conf/
 
 # step on default config with our redhat one
-cp -p redhat/%{username}.in.sh %{buildroot}/usr/share/%{username}/%{username}.in.sh
-cp -p redhat/%{username} %{buildroot}/%{_sysconfdir}/rc.d/init.d/%{username}
-cp -p redhat/%{username}.conf %{buildroot}/%{_sysconfdir}/security/limits.d/
```

Comment:
Wasn't it important to configure ulimit ?

Reply:
AFAIK systemd ignores PAM limits anyway). We set the ulimit for Cassandra's systemd unit [here](https://github.com/aiven/aiven-core/blob/0dbd507885f4f6e62dd5a226aeac4801e9458fc4/aiven/prune/services/cassandra/cassandra.py#L493)

Reply:
Ok, yes it seem to be ignored:
https://shellpower.wordpress.com/2019/11/28/systemd-not-honoring-etc-security-limits-conf/